content/en/tracing/trace_collection/otel_instrumentation/ruby.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Custom Instrumentation of Ruby Applications with the OpenTelemetry API
+kind: documentation
+description: 'Instrument your Ruby application with OTel API to send traces to Datadog.'
+code_lang: ruby
+type: multi-code-lang
+code_lang_weight: 20
+further_reading:
+    - link: 'tracing/glossary/'
+      tag: 'Documentation'
+      text: 'Explore your services, resources, and traces'
+---
+
+{{% otel-custom-instrumentation %}}
+
+
+## Requirements and limitations
+
+- Datadog Ruby tracing library `dd-trace-rb` version 1.9.0 or greater
+- Gem version support 1.1.0 or greater
+
+The following OTel features implemented in the Datadog library as noted:
+
+| Feature                               | Support notes                       |
+|---------------------------------------|-------------|-----------------------------|
+| [OTel Context propagation][1]         | [Datadog distributed header format][9] is used instead. | 
+| [Span processors][2]                  | Unsupported                                          | 
+| [Span Exporters][3]                   | Unsupported                                            |
+| `OpenTelemetry.logger`                | `OpenTelemetry.logger` is set to the same object as `Datadog.logger`. Configure through [custom logging][10]. |
+| Trace/span [ID generators][4]         | ID generation is performed by `ddtrace`.           |
+
+
+## Configuring OTel to use the Datadog tracing library
+
+1. Add your desired manual OTel instrumentation to your Ruby code following the [OTel Ruby Manual Instrumentation documentation][5].
+1. Add the `ddtrace` gem to your Gemfile:
+
+    ```ruby
+    source 'https://rubygems.org'
+    gem 'ddtrace'
+    ```
+
+1. Install the gem by running `bundle install`.
+1. Add the following lines to your OpenTelemetry configuration file:
+
+    ```ruby
+    require 'opentelemetry'
+    require 'datadog/opentelemetry'
+    ```
+
+1. Add a configuration block to your application where you can activate integrations and change tracer settings. Without additional configuration here, only code you have instrumented with OTel is traced:
```

Comment:
We might need to make this change in a few places
```suggestion
1. Add your desired manual OTel instrumentation to your Ruby code following the [OTel Ruby Manual Instrumentation documentation][5].
2. Add the `ddtrace` gem to your Gemfile:

    ```ruby
    source 'https://rubygems.org'
    gem 'ddtrace'
    ```

3. Install the gem by running `bundle install`.
4. Add the following lines to your OpenTelemetry configuration file:

    ```ruby
    require 'opentelemetry'
    require 'datadog/opentelemetry'
    ```

4. Add a configuration block to your application where you can activate integrations and change tracer settings. Without additional configuration here, only code you have instrumented with OTel is traced:
```

Reply:
Markdown is smart and handles the numbering correctly with all `1.`